gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -1101,7 +1113,7 @@ gst_tensor_trainer_get_tensor_size (GstTensorTrainer * trainer,
     return 0;
   }
 
-  return gst_tensor_info_get_size (&info->info[index]);
+  return gst_tensor_info_get_size (gst_tensors_info_get_nth_info (info, index));
```

Comment:
recommend using gst_tensors_info_get_size (info, index);

Reply:
Thank you !!